build/avalanchego-apis/admin-api.md

```diff
@@ -106,17 +106,17 @@ Now, instead of interacting with the blockchain whose ID is `sV6o671RtkGBcno1Fia
 
 ### admin.lockProfile
 
-Writes a profile of mutex statistics to `lock.profile`.
+Writes a profile of mutex statistics a file named `lock.profile` in the home directory.
```

Comment:
`Writes a profile of mutex statistics a file named `lock.profile` in the home directory.` --> `Writes a profile of mutex statistics to a file named `lock.profile` in the home directory.`